schemas/context/segmentmembership.schema.json

```diff
@@ -29,6 +29,67 @@
           "format": "date-time",
           "description": "The timestamp when the assertion of segment membership was made."
         },
+        "xdm:payload" : {
+          "title" : "Payload",
+          "description": "Values that are directly related with the segment realization. This payload exists with the same validUntil as the segment realization.",
+          "oneOf": [
+            {
+              "type": "object",
+              "properties": {
+                "xdm:payloadNumberValue": {
```

Comment:
We could support only number type first. We'll punt on the other types. 

Create an issue for polymorophic types using oneOf and const types. This against adobe/xdm repo. cc: Kevin